lib/games_methods.rb

```diff
@@ -13,6 +13,10 @@ def create_games(csv_file_path)
        Game.new(row)
     end
   end
+
+  def all
```

Comment:
made this pup to avoid things like `games.games` so now its game.all

Reply:
Nice fix. Less confusion